probatus/feature_elimination/feature_elimination.py

```diff
@@ -354,14 +370,23 @@ def fit(self, X, y, column_names=None):
         if self.random_state is not None:
             np.random.seed(self.random_state)
 
+        if columns_to_keep is None:
+            columns_to_keep = []
+        
+
```

Comment:
Could you make a check if the columns_to_keep is a list of string that match the X column names?

Reply:
Done.